ui/tests/e2e/custom-playwright-reporter.js

```diff
@@ -6,39 +6,37 @@ import path from 'path';
 class MyReporter {
   introMessage = '';
   totalTests = '';
-  testData = [];
+  expectedTest = '';
+  testTable = `| Test | Browser | Test Case | Tags | Result |
```

Comment:
will this change be efficient from performance perspective if number of test results increases?

Reply:
Hey team thanks for the reviews. 🙏

To give some context, this string usage was the test reporter structure that was present prior to a change 2 weeks ago trying to make this integrate with the docs site. Let me know if you wish to tackle improvements in the original reporter structure in this same PR. 

I would prefer to keep thee changes separated but let me know your pov. Thank you 🚀